scripts/milestone.py

```diff
@@ -0,0 +1,336 @@
+#!/usr/bin/env python3
+
+# Standard library imports
+import datetime
+import logging
+import os
+import sys
+from itertools import groupby
+from pathlib import Path
+
+# External imports
+import click
+import requests
+
+VALID_TYPES = [
+    "type: bug",
+    "type: feature",
+    "type: task",
+]
```

Comment:
nit: I prefer tuples for "constants" like this

Reply:
Yah that seems reasonable here 👍